h5py/tests/common.py

```diff
@@ -66,35 +63,33 @@ def mktemp(self, suffix='.hdf5', prefix='', dir=None):
         if dir is None:
             dir = self.tempdir
         return tempfile.mktemp(suffix, prefix, dir=self.tempdir)
-        
+
     def setUp(self):
         self.f = h5py.File(self.mktemp(), 'w')
-        
+
     def tearDown(self):
         try:
             if self.f:
                 self.f.close()
         except:
             pass
 
-    if not hasattr(ut.TestCase, 'assertSameElements'):
-        # shim until this is ported into unittest2
-        def assertSameElements(self, a, b):
-            for x in a:
-                match = False
-                for y in b:
-                    if x == y:
-                        match = True
-                if not match:
-                    raise AssertionError("Item '%s' appears in a but not b" % x)
-
-            for x in b:
-                match = False
-                for y in a:
-                    if x == y:
-                        match = True
-                if not match:
-                    raise AssertionError("Item '%s' appears in b but not a" % x)
+    def assertSameElements(self, a, b):
```

Comment:
So this appeared in python2.7 (or at least I think it did, 2.7 made a whole bunch of changes to unittest, resulting in unittest2 to backport changes, which has kept up with unittest changes, so current unittest2 matches python 3.6), was deprecated in python 3.2, and dropped in python 3.3, so it's simpler if we just use our custom version across all versions (I considered replacing this function entirely, but that would complicate this patch, which was mainly about removing the duplication).